tools/ValidateInputFile.py

```diff
@@ -134,7 +134,9 @@ def validate_input_file(
         input_file_path=input_file_path,
         line_number=line_number,
         yaml_path=path,
-        message="\n".join(msg).strip(),
+        message=(
+            "\n".join(msg).strip() if found_hints else process.stderr.strip()
+        ),
```

Comment:
Actually wait, will this print anything? The `render_context` in `InvalidInputFileError` is empty without a line number and yaml path (neither of which exist if `found_hints` is false)

Reply:
The exception message will print. `render_context` is just extra stuff.

Reply:
Ok cool. I can't test this now because I fixed the issue in the code that was causing an FPE during option parsing, but hopefully next time I hit something this will show up.

Reply:
This also happens when you put a wrong `Version` at the top of the file